plugins/sentinelone/bin/komand_sentinelone

```diff
@@ -6,7 +6,7 @@ from sys import argv
 
 Name = "SentinelOne"
 Vendor = "rapid7"
-Version = "9.2.0"
+Version = "10.0.0"
 Description = "The SentinelOne plugin allows you to manage and mitigate all your security operations through SentinelOne"
```

Comment:
Think we need to bump the connection version aswell or was that done in the last PR?